src/d_main.c

```diff
@@ -319,6 +314,10 @@ void D_Display (void)
   switch (gamestate)                // do buffered drawing
     {
     case GS_LEVEL:
+      if (uncapped && !menuactive && !paused && raw_input)
+      {
+        I_StartDisplay();
```

Comment:
So it's literally a few lines later? I prefer a clear code structure in cases like this. Perhaps someday we will reformat and refactor this whole file.

Reply:
Yeah I thought the same but if you look at the disassembly in VS there's a lot of intervening code.

Reply:
Maybe, but it's probably looks different in other compiler.

If you run Woof in profiler (or any Doom engine with software rendering), you'll see that it mostly spends its time in the `R_DrawColumn` function. For this function, I think we can live with it being written entirely in assembler or with a lot of intrinsics. For `D_Display` the difference is small, I would prefer it to be more readable.